frontend/packages/dev-console/src/actions/modify-helm-release.ts

```diff
@@ -0,0 +1,20 @@
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { history } from '@console/internal/components/utils';
+import { deleteResourceModal } from '../components/modals';
+
+export const deleteHelmRelease = (releaseName: string, namespace: string, redirect?: string) => {
+  return {
+    label: 'Delete Helm Release',
+    callback: () => {
+      deleteResourceModal({
+        blocking: true,
+        resourceName: releaseName,
+        resourceType: 'Helm Release',
+        onSubmit: () => {
+          return coFetchJSON.delete(`/api/helm/release?name=${releaseName}&ns=${namespace}`);
```

Comment:
What happens if the network call fails? Where does the error go?

Reply:
Updated the modal to show the error inline. Earlier it was showing the error but was also closing the modal immediately after.